schema/changelog-3.10.xml

```diff
@@ -48,5 +48,21 @@
       <column name="network" type="VARCHAR(4000)" />
     </addColumn>
 
+    <addColumn tableName="users">
+      <column name="userlimit" type="INT" defaultValueNumeric="0" />
+    </addColumn>
+
+    <createTable tableName="user_user">
+      <column name="userid" type="INT">
+        <constraints nullable="false" />
+      </column>
+      <column name="otheruserid" type="INT">
```

Comment:
The name is a bit confusing. Maybe it's better to name it `manageduserid` or something else. Can't really think of any good name.